test/acceptance/behave/components/ngsiv2/entities/create_entities.feature

```diff
@@ -0,0 +1,1899 @@
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+#
+# author: 'Iván Arias León (ivan dot ariasleon at telefonica dot com)'
+
+#
+#  Note: the "skip" tag is to skip the scenarios that still are not developed or failed
+#        -tg=-skip
+#
+
+# Missing Tests:
+#   - verification of Maximum Length Allowed in Fields
+#   - verification of Special Attribute Types
+#
+
+
+Feature: create entities requests (POST) in NGSI v2
+  As a context broker user
+  I would like to  create entities requests in NGSI v2
+  So that I can manage and use them in my scripts
+
+  BackgroundFeature:
+  Setup: update properties test file from "epg_contextBroker.txt" and sudo local "false"
+  Setup: update contextBroker config file and restart service
+  Check: verify contextBroker is installed successfully
+  Check: verify mongo is installed successfully
+
+  @happy_path
+  Scenario:  create several entities in NGSI v2
+    Given  a definition of headers
+      | parameter          | value            |
+      | Fiware-Service     | test_happy_path  |
+      | Fiware-ServicePath | /test            |
+      | Content-Type       | application/json |
+    When create "3" entities with "2" attributes
+      | parameter        | value                   |
+      | entities_type    | room                    |
+      | entities_id      | room2                   |
+      | attributes_name  | random=5                |
+      | attributes_value | 017-06-17T07:21:24.238Z |
+      | attributes_type  | date                    |
+      | metadatas_number | 2                       |
+      | metadatas_name   | very_hot                |
+      | metadatas_type   | alarm                   |
+      | metadatas_value  | hot                     |
+    Then verify that receive several "Created" http code
+    And verify that entities are stored in mongo
+
+  @maximum_size
+  Scenario:  try to create a new entity NGSI v2 with maximun size in payload (8972 attributes = 1048576 bytes)
+    Given  a definition of headers
+      | parameter          | value             |
+      | Fiware-Service     | test_maximum_size |
+      | Fiware-ServicePath | /test             |
+      | Content-Type       | application/json  |
+    When create "1" entities with "8972" attributes
+      | parameter        | value     |
+      | entities_type    | room      |
+      | entities_id      | room2     |
+      | attributes_name  | max_size  |
+      | attributes_value | random=10 |
+      | attributes_type  | my_type   |
+      | metadatas_number | 1         |
+      | metadatas_name   | very_hot  |
+      | metadatas_type   | alarm     |
+      | metadatas_value  | random=10 |
+    Then verify that receive several "Request Entity Too Large" http code
+    And verify several error responses
+      | parameter   | value                                              |
+      | error       | Request Entity Too Large                           |
```

Comment:
Now this test does not fail as it is conceived. Example:
With a payload length = 1048647 in request
Returns in response:
```
 http code: 413
{"error":"Request Entity Too Large","description":"payload size: 1048647, max size supported: 1048576"}
```